core/game.py

```diff
@@ -11,10 +12,12 @@ class Game:
     formulation.
     """
 
-    def __init__(self, starting_layout: BoardLayout = BoardLayout.GERMAN_DAISY):
+    def __init__(self, starting_layout: BoardLayout = config.Config.DEFAULT_LAYOUT):
         self._board = BoardLayout.setup_board(starting_layout)
         self._turn = Color.BLACK
 
+        self.temporary_move_count = [0, 0]  # Will be removed when history is implemented
```

Comment:
Temporary store for move count, will remove when History is implemented, as it can be derived from that.